config/crd/bases/argoproj.io_argocds.yaml

```diff
@@ -2211,72 +2238,1254 @@ spec:
                     description: ServiceAccount defines the ServiceAccount user that
                       you would like the Repo server to use
                     type: string
-                  verifytls:
-                    description: VerifyTLS defines whether repo server API should
-                      be accessed using strict TLS validation
-                    type: boolean
-                  version:
-                    description: Version is the ArgoCD Repo Server container image
-                      tag.
-                    type: string
-                  volumeMounts:
-                    description: VolumeMounts adds volumeMounts to the repo server
-                      container
+                  sidecarContainers:
```

Comment:
This is the argocd CRD from the master branch of the argocd-operator, not the one from the release-0.3 branch.

Reply:
Ack, made required changes.